packages/react/scroll-area/package.json

```diff
@@ -25,7 +25,7 @@
     "@radix-ui/react-polymorphic": "workspace:*",
     "@radix-ui/react-primitive": "workspace:*",
     "@radix-ui/react-use-callback-ref": "workspace:*",
-    "@radix-ui/react-use-layout-effect": "workspace:*"
+    "@radix-ui/react-use-direction": "workspace:*"
   },
   "devDependencies": {
     "@stitches/react": "canary",
```

Comment:
Not sure why this was here but it can go, considering we're in a monorepo, we have it at the top-level.